rllib/algorithms/ppo/ppo.py

```diff
@@ -456,7 +455,7 @@ def training_step(self) -> ResultDict:
                 # TODO (Kourosh): Train results don't match the old format. The thing
                 # that used to be under `kl` is now under `mean_kl_loss`. Fix this. Do
                 # we need get here?
-                pid: train_results["loss"][pid].get("mean_kl_loss")
+                pid: train_results[pid][LEARNER_STATS_KEY].get("mean_kl_loss")
```

Comment:
If I want to nit picking, this change is actually changing a public external spec.
for example, people's W&B charts, or analysis scripts will break once we change the key.
we of course should do this, but it's gonna be messy to keep the change behind a flag right?

Reply:
we were breaking it before. Now it's back to the agreement we had on the old stack.